spec/country_select_spec.rb

```diff
@@ -77,11 +77,23 @@ class Walrus
     expect(t).to include(tag)
   end
 
-  it "selects only the first matching option" do
-    tag = options_for_select([["United States", "US"],["Uruguay", "UY"]], "US")
-    walrus.country_code = 'US'
-    t = builder.country_select(:country_code, priority_countries: ['LV','US'])
-    expect(t).to_not include(tag)
+  describe "when selected options is not an array" do
+    it "selects only the first matching option" do
+      tag = options_for_select([["United States", "US"],["Uruguay", "UY"]], "US")
+      walrus.country_code = 'US'
+      t = builder.country_select(:country_code, priority_countries: ['LV','US'])
+      expect(t).to_not include(tag)
+    end
+  end
+
+  describe "when selected options is an array" do
+    it "selects all options but only once" do
+      tag = options_for_select([["United States", "US"],["Uruguay", "UY"],["Spain", "ES"]], "US")
+      walrus.country_code = 'US'
+      t = builder.country_select(:country_code, priority_countries: ['LV','US','ES'], selected: ['UY', 'US'])
```

Comment:
I'm using [chosen-rails gem](https://github.com/tylergannon/chosen-rails) and of course, with `multiple: true` option. Then in model `serialize :origin_country, Array` in order to use just one property